src/orca-loader/OrcaUtils.res

```diff
@@ -330,3 +331,30 @@ let getArrayOfTupleFromDict = dict => {
   ->Js.Dict.keys
   ->Belt.Array.map(key => (key, Js.Dict.get(dict, key)->Belt.Option.getWithDefault(Js.Json.null)))
 }
+
+let makeOneClickHandlerPromise = sdkHandleOneClickConfirmPayment => {
```

Comment:
Try without using recursive to remove the event listener, otherwise could lead to intermittent issues.